src/data_generators/m3_data_design.py

```diff
@@ -91,32 +92,40 @@
     GeneratorParameters("HeavyTailed", HeavyTailedImpressionGenerator, {"zeta_s": 5.0}),
 ]
 
-# TODO(jiayu):
-# 1. Add more Copula generators once we decide the parameters (say,
-# the Gaussian correlation matrix.)
-# 2. Remove some of the Independent & Sequential overlap_generators
-# So we have a higher fraction of Copula generators.
-OVERLAP_GENERATORS = [
+RNG = np.random.default_rng(0)
+OVERLAP_GENERATORS_INDEPENDENT_GIVEN_UNIVERSE_SIZE = [
     GeneratorParameters("FullOverlap", DataSet, {}),
     GeneratorParameters(
         "Independent",
         IndependentOverlapDataSet,
-        {"largest_pub_to_universe_ratio": 0.9, "random_generator": 1},
+        {
+            "largest_pub_to_universe_ratio": 0.9,
```

Comment:
Is there a relationship between this value and the values given in PUBLISHER_RATIOS at line 58?

Reply:
Sure, added!